base/pkg/entry.jl

```diff
@@ -380,15 +380,14 @@ function update(branch::AbstractString, upkgs::Set{String})
                 Pkg.dir("METADATA") * ".", cex))
         end
     end
-    deferred_errors = CompositeException()
+    deferred_warnings = String[]
     avail = Read.available()
     # this has to happen before computing free/fixed
     for pkg in filter(Read.isinstalled, collect(keys(avail)))
         try
             Cache.prefetch(pkg, Read.url(pkg), [a.sha1 for (v,a)=avail[pkg]])
         catch err
-            cex = CapturedException(err, catch_backtrace())
-            push!(deferred_errors, PkgError("Package $pkg: unable to update cache.", cex))
+            push!(deferred_warnings, string("Package $pkg: unable to update cache\n    └ ", err, "\n"))
```

Comment:
Is that character commonly available in terminals? IIRC the Windows terminal has some issues with certain box drawing characters.